docker-compose.yml

```diff
@@ -99,17 +99,16 @@ services:
   ## Grafana / Prometheus
 
   grafana-server:
-    image: grafana/grafana:5.2.4
+    build: ./docker/grafana
     container_name: grafana-server
     mem_limit: 256M
     ports:
     - 3000:3000
-    volumes:
-    - graf-data:/var/lib/grafana
+    ##volumes:
```

Comment:
since these aren't necessary anymore, i'd recommend just leaving it out

Reply:
Nice hook. Fix -)